chapters/en/slides/module7/module7_04.Rmd

```diff
@@ -0,0 +1,342 @@
+---
+params:
+  dynamictitle: "module7_04"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+
+# Sharing Altair Visualizations
+
+Notes:
+In this module
+we will learn how we can share our visualizations 
+with others,
+both online and offline.
+
+---
+
+## Charts are easily saved as HTML files
+
+```python
+import altair as alt
+from vega_datasets import data
+
+cars = data.cars.url
+chart = alt.Chart(cars).mark_circle().encode(
+    alt.X('Horsepower:Q'),
+    alt.Y('Miles_per_Gallon:Q'))
+
+chart.save('cars-scatterplot.html')
+```
+
+
+Notes:
+The easiest way to save Altair charts is as HTML document
+via the `save` method as we have done in this slide.
+This file an either be incorporated in a web page 
+or sent to anyone via e.g. email.
+
+When they open the file in a web browser
+they will see the visualization,
+including any interactivity you have used!
+
+This is all you need to know for sharing your visualizations
+with others outside the Jupyter Notebook,
+but read on for more technical details about how this work
+and how you can save to other formats than HTML.
+
+---
+
+## How are Altair visualization rendered?
+
+- Altair does not create images to represent charts
+- Instead if create Vega-Lite code
+- Your browser knows how to render Vega-Lite code as graphics
+
```

Comment:
I can't think of a nice way of doing this. In the HTML there is essentially one line of text with the entire vega lite spec which almost maps one to one to the altair code, so there are no other elements on the web page to points to with arrows. In our example it would just be x-axis, y-axis, and circle mark that shows up on the page.